Scripts/assignment.py

```diff
@@ -51,26 +51,22 @@ def assign(self, scen_id, time_period, matrices):
         """Assign cars, bikes and transit for one time period."""
         self.logger.info("Assignment starts...")
         self.set_matrices(matrices)
-        function_file = os.path.join(self.path,
-                                     "d411_pituusriippuvaiset_HM30.in")
-        process = self.emme_modeller.tool(
-            "inro.emme.data.function.function_transaction")
-        process(function_file)
         self.calc_road_cost(scen_id)
         self.assign_cars(scen_id, param.stopping_criteria_coarse)
+        self.calc_boarding_penalties(scen_id)
```

Comment:
We could consider if we want to use python convention of private methods with prefix self.__method_name() instead of self.method_name(). This could make it more intuitive which methods are considered to be part of the public interface of this class, == which methods should I use to process some data.